src/main/kotlin/org/rust/lang/core/stubs/StubInterfaces.kt

```diff
@@ -6,16 +6,58 @@
 package org.rust.lang.core.stubs
 
 import com.intellij.util.BitUtil
-import org.rust.lang.core.psi.ext.QueryAttributes
-import org.rust.lang.core.psi.ext.RsDocAndAttributeOwner
-import org.rust.lang.core.psi.ext.getTraversedRawAttributes
-import org.rust.lang.core.psi.ext.name
+import org.rust.lang.core.psi.ext.*
+import org.rust.lang.core.resolve.KNOWN_DERIVABLE_TRAITS
 import org.rust.stdext.BitFlagsBuilder
+import org.rust.stdext.HashCode
 
 interface RsNamedStub {
     val name: String?
 }
 
+/**
+ * A common interface for stub for elements that can hold attribute or derive procedural macro attributes
```

Comment:
Is it supposed to be used for attribute macro support? If yes, let's make doc comment more general. Otherwise, we will forget to do it

Reply:
It's generic already:

A common interface for stub for elements that can hold **attribute** or derive procedural macro attributes